src/openrct2/drawing/Drawing.String.cpp

```diff
@@ -20,6 +21,83 @@
 
 #include <algorithm>
 
+using namespace OpenRCT2;
+
+class CodepointView
+{
+private:
+    std::string_view _str;
```

Comment:
Don't use use TitleCase for member variables?

Reply:
This isn't title case.